runtime/service.go

```diff
@@ -315,6 +315,7 @@ func (s *service) StartShim(shimCtx context.Context, containerID, containerdBina
 		}
 
 		s.vmID = uuid.String()
+		s.logger = s.logger.WithField("vmID", s.vmID)
```

Comment:
Since we are adding this, we now longer need to include `s.vmID` in the logs written on lines 325 and 327 below.